docs/integrations/sources/google-webfonts.md

```diff
@@ -0,0 +1,68 @@
+# Google-webfonts
+
+This page contains the setup guide and reference information for the [Google-webfonts](https://developers.google.com/fonts/docs/developer_api) source connector.
+
+## Prerequisites
+
+Api key is mandate for this connector to work, It could be generated by a gmail account for free at https://console.cloud.google.com/apis/dashboard.
+Just pass the generated API key and optional parameters for establishing the connection. Example:123
+
+## Setup guide
+
+### Step 1: Set up Google-webfonts connection
+
+- Generate an API key (Example: 12345)
+- Params (If specific info is needed)
+- Available params
+  - sort: SORT_UNDEFINED, ALPHA, DATE, STYLE, TRENDING, POPULARITY
+  - alt: json, media or proto
+  - prettyPrint: boolean
+
+## Step 2: Set up the Google-webfonts connector in Airbyte
+
+### For Airbyte Cloud:
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+new source**.
+3. On the Set up the source page, enter the name for the Google-webfonts connector and select **Google-webfonts** from the Source type dropdown.
+4. Enter your `api_key`.
+5. Enter the params configuration if needed. (Optional)
```

Comment:
Can you be more specific with the params? Can you add the `sort`, `alt`, `prettyPrint` here?

Reply:
Done